test/specs/TC5_About_link.js

```diff
@@ -0,0 +1,19 @@
+import loginPage from '../pageobjects/login.page.js'
+import inventoryPage from '../pageobjects/inventory.page.js'
+import { config } from '../../config.js';
+const { username, password } = config.valid_user;
+
+describe('TC5 - About link test', () => {
+    before(async () => {
+        await loginPage.open()
+        await loginPage.login(username, password)
+    });
+
+    it('should check that About link opens', async () => {
+        await inventoryPage.clickBurger()
+        await browser.pause(1000)
```

Comment:
@karenn32 If you added somethink lige that - sometimes add the comment, to explain, why is it here.